wd_capabilities.au3

```diff
@@ -12,10 +12,11 @@
 ; Title .........: wd_capabilities.au3
 ; AutoIt Version : v3.3.14.5
 ; Language ......: English
-; Description ...: A collection of functions used to dynamically build the Capabilities string required to create a WebDriver session
+; Description ...: A collection of functions used to dynamically build the Capabilities string (JSON formated) required to create a WebDriver session
```

Comment:
The correct spelling in English is "formatted". See https://wikidiff.com/formated/formatted for the difference.

Reply:
fixed:
[capability case sensitivity](https://github.com/Danp2/au3WebDriver/pull/285/commits/58b8ba22aa5e8571c5908ca1f137dc0384f20827)